packages/forms/resources/views/components/date-time-picker.blade.php

```diff
@@ -1,13 +1,10 @@
 @once
     @push('scripts')
         <script src="//unpkg.com/dayjs@1.10.4/dayjs.min.js"></script>
```

Comment:
Do we still need the CDN version too?

Reply:
I did a test removing the cdn call from dayjs.min.js and the DatePicker still works but the translations don't load.

Reply:
And does this work outside the admin panel also?

Reply:
I tested with admin and without it
Removing the localeData cdn didn't cause any problem but for some reason the translations stop working when  remove the dayjs cdn .

I think this would need further investigation as I think it's redundant to keep this cdn when it's included in the bundle.